README.md

```diff
@@ -49,7 +51,14 @@ Options:
   -h, --help                         display help for command
 ```
 
-# Installation for Developers
+# GoFSH Developer Instructions
+
+## Intro to GoFSH Development
+
+To learn more about GoFSH, watch the Knowledge Sharing Sessions for [Developing FSH Tools](https://vimeo.com/990594107/c35c3f5e43) (view the slides [here](https://confluence.hl7.org/display/FHIR/FSH+Knowledge+Sharing+Sessions?preview=/256509612/256514908/KSS%203%20-%20Developing%20FSH%20Tools.pdf)) and [Developing GoFSH](https://vimeo.com/1001309480/f2b262aee7) (view the slides [here](https://confluence.hl7.org/display/FHIR/FSH+Knowledge+Sharing+Sessions?preview=/256509612/265487253/KSS%205%20-%20Developing%20GoFSH.pdf)).
```

Comment:
As noted in the corresponding SUSHI PR, the Developing FSH Tools link unfortunately points to the wrong video. This should fix it.
```suggestion
To learn more about GoFSH, watch the Knowledge Sharing Sessions for [Developing FSH Tools](https://vimeo.com/990594228/056b5c075f) (view the slides [here](https://confluence.hl7.org/display/FHIR/FSH+Knowledge+Sharing+Sessions?preview=/256509612/256514908/KSS%203%20-%20Developing%20FSH%20Tools.pdf)) and [Developing GoFSH](https://vimeo.com/1001309480/f2b262aee7) (view the slides [here](https://confluence.hl7.org/display/FHIR/FSH+Knowledge+Sharing+Sessions?preview=/256509612/265487253/KSS%205%20-%20Developing%20GoFSH.pdf)).
```

Reply:
Fixed in [4e36bcf](https://github.com/FHIR/GoFSH/pull/269/commits/4e36bcf35d0ea0262d41a68daa578cc5402768dc)